Azure/azure-cli
# [RDBMS] `az mysql flexible-server ad-admin delete`: Disable `aad_auth_only` when dropping AAD admin

**Related command**
`az mysql flexible-server ad-admin delete`

**Description**
When dropping the AAD admin for a MySQL flexible sever, we should disable automatically **AAD authentication only** because customer won't be able to connect again to the server using username and password. He can disable it manually by setting `aad_auth_only` to `OFF` and connect again, but the idea is to do this automatically to avoid confusion.

**Testing Guide**
- Create an AAD admin for an existing server: `az mysql flexible-server ad-admin create -g resourceGroup -s serverName -u login -i sid --identity userIdentity`
- Enable AAD authentication only: `az mysql flexible-server parameter set -g resourceGroup -s serverName -n aad_auth_only -v ON`
- Delete the AAD admin: `az mysql flexible-server ad-admin delete -g resourceGroup -s serverName`
- Verify that server has MySQL authentication only (`aad_auth_only` should be `OFF`): `az mysql flexible-server parameter show -g resourceGroup -s serverName -n aad_auth_only`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
